### DISCUSSION TITLE
Execute nextflow script on k8s cluster with Minio instead of AWS S3

### DISCUSSION THREAD

**Discussion by dulguunb** [nextflow-io/nextflow#3402] [Q&A]
I couldn't find any documentation about executing nextflow pipeline on the k8s cluster using S3 compatible storage systems e.g: Minio. Is it possible to execute nextflow pipeline like this?
I have been trying to execute it like this `nextflow run nextflow-test1/minimal -entry step1 https://<minio_host>:9000/nextflow-test` but it hasn't been successful so far

**Comment by pditommaso** [comment]
Using Minio is not possible right now, currently we are supporting this use case with S3 using Wave containers. See [here](https://www.nextflow.io/docs/latest/wave.html) and [here](https://github.com/seqeralabs/wave-showcase/tree/master/example8). 

We plan to support Minio soon.

**Reply by dulguunb** [reply]
Ah, okay, thank you.

**Reply by pditommaso** [reply]
Follow this issue for updates https://github.com/nextflow-io/nextflow/issues/3406
